app/views/provider_interface/references/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<% content_for :browser_title, "#{@application_choice.application_form.full_name} – #{@application_choice.course.name_and_code} - References" %>
+
+<%= render ProviderInterface::ApplicationChoiceHeaderComponent.new(
+  application_choice: @application_choice,
+) %>
+<h1 class="govuk-heading-l">References</h1>
+
+<% if ApplicationStateChange::OFFERED_STATES.exclude? @application_choice.status.to_sym %>
```

Comment:
Isn't this conditional the same as `no_offer_yet?` could this be a method on `ApplicationChoice`? Perhaps taking into account other comments about the method name itself.